k8s/helm/templates/ingress.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.ingress.enabled -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Any reason this is still using the beta and not `networking.k8s.io/v1`?.

Reply:
As per the k8s docs:
```
The v1.22 release will stop serving the following deprecated API versions in favor of newer and more stable API versions:

Ingress in the extensions/v1beta1 API version will no longer be served
Migrate to use the networking.k8s.io/v1beta1 API version, available since v1.14. Existing persisted data can be retrieved/updated via the new version.
```

We don't actually _need_ to upgrade now (it isn't deprecated till `v1.22`, but we might as well do it while we aren't running the prod service?

Reply:
Given we are updating it here, I think we should only use the beta if we can't use `v1` which I don't think is the case.

Reply:
As far as I can see, the docs say to use v1beta1 for ingress - is there other documentation that states otherwise?

Reply:
v1 is not available until our cluster is on 1.19